src/main/java/com/github/peggybrown/speechrank/InMemoryConferencesRepository.java

```diff
@@ -42,7 +36,7 @@ public void add(Rate rate) {
     }
 
     @Override
-    public void add(Comment comment) {
+    public void add(Comment comment){
```

Comment:
please restore the old formatting with a single space after methods arguments list